crates/ironrdp-dvc/Cargo.toml

```diff
@@ -13,14 +13,17 @@ categories.workspace = true
 
 [lib]
 doctest = false
-test = false
 
 [features]
 default = []
 std = []
 
 [dependencies]
+bitflags.workspace = true
 ironrdp-svc.workspace = true
 ironrdp-pdu = { workspace = true, features = ["alloc"] }
 tracing.workspace = true
 slab = "0.4.9"
+
+[dev-dependencies]
+lazy_static = "1.4"
```

Comment:
suggestion: Don’t use `lazy_static` anymore. We have [`OnceLock` in the standard library now](https://doc.rust-lang.org/std/sync/struct.OnceLock.html).